src/postgres-util/Cargo.toml

```diff
@@ -8,6 +8,7 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.55"
+mz-dataflow-types = { path = "../dataflow-types" }
```

Comment:
This dependency edge looks backwards to me! I think either move the protobufs into `postgres-util` (sorry, I think I might have said the wrong thing earlier), or move the `From` impls into the `dataflow-types` crate as `Into` impls, and invert the dependency.

No clear opinion on which of these options is better, though moving the protobufs into `postgres-util` has the maybe appealing property that you can just entirely delete the `PgColumn` and `TableInfo` structs and just use the protobuf types directly.

Reply:
`PgColumn` and `TableInfo` are used in multiple other places so this would be a more invasive change, and when we originally decided to put it in `dataflow-types` it was because conceptually the protobufs are directly related to the postgres dataflow source and not postgres generically.

Would it not work to move the `From` impls into `dataflow-types` and then invert the dependency?

Reply:
> Would it not work to move the From impls into dataflow-types and then invert the dependency?

It would! That was one of the options I mentioned above! You might have to make them `Into`s instead, but should totally work.